src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -924,8 +930,8 @@ Additional information: {1}</value>
   <data name="Options" xml:space="preserve">
     <value>Options</value>
   </data>
-  <data name="Try_the_preview_version_of_our_live_code_analysis_extension_which_provides_more_fixes_for_common_API_design_naming_performance_and_reliability_issues" xml:space="preserve">
-    <value>Try the preview version of our live code analysis extension, which provides more fixes for common API design, naming, performance, and reliability issues</value>
+  <data name="Install_Microsoft_recommended_Roslyn_analyzers_which_provide_additional_diagnostics_and_fixes_for_common_API_design_security_performance_and_reliability_issues" xml:space="preserve">
+    <value>Install Microsoft recommended Roslyn analyzers, which provide additional diagnostics and fixes for common API design, security, performance, and reliability issues</value>
```

Comment:
Do we want to explicitly mention FxCop somewhere in this text? It might be a keyword that can catch existing FxCop users' attention.

Reply:
We can consider it, but it seems to cause lot of confusion to users who feel legacy binary analysis is `FxCop` and new analysis is `Roslyn analyzers`. For example, see https://developercommunity.visualstudio.com/content/problem/635683/incorrect-text-run-code-analysis-has-been-deprecat.html.

Our primary goal here is to help _new users_ discover and enable additional live code analysis. Most of the legacy FxCop users should have moved to FxCopAnalyzers, and the ones that haven't are due to their migration priorities, and unlikely to install the analyzers anyway.

Also note that `Learn More` hyperlink takes the users directly to FxCopAnalyzers install page: https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019